tests/csm/rest/test_bucket.py

```diff
@@ -160,12 +169,10 @@ def test_578(self):
         assert_utils.assert_equals(response.status_code,
                                    const.BAD_REQUEST)
         if CSM_REST_CFG["msg_check"] == "enable":
-            assert_utils.assert_equals(response.json()["error_code"],
-                                    str(resp_error_code))
-            assert_utils.assert_equals(response.json()["message"],
-                                  msg)
-        assert_utils.assert_equals(response.json()["message_id"],
-                                  resp_msg_id)
+            self.log.info("Verifying error response...")
+            assert_utils.assert_equals(response.json()["error_code"], resp_error_code)
```

Comment:
Can u also log error message  "Error code mismatch" in case of assertion

Reply:
as assert_equals printing both the values, so skipping.